Common/include/CConfig.hpp

```diff
@@ -3728,6 +3798,29 @@ class CConfig {
    */
     su2double GetThermal_Conductivity_ConstantND(void) const { return Thermal_Conductivity_ConstantND; }
 
+  /*!
+   * \brief Get the value of the constant mass diffusivity for scalar transport.
+   * \return Constant mass diffusivity.
+   */
+  su2double GetDiffusivity_Constant(void) const { return Diffusivity_Constant; }
+
+  /*!
+   * \brief Get the value of the non-dimensional constant mass diffusivity.
+   * \return Non-dimensional constant mass diffusivity.
+   */
+  su2double GetDiffusivity_ConstantND(void) const { return Diffusivity_ConstantND; }
+
+  /*!
+   * \brief Get the value of the laminar Schmidt number for scalar transport.
+   * \return Laminar Schmidt number for scalar transport.
+   */
+  su2double GetSchmidt_Number_Laminar(void) const { return Schmidt_Number_Laminar; }
```

Comment:
Laminar Schmidt number is currently unused. I would have to implememt/copy a Schmidt-based diffusivity model, which is somewhat straight forward but we also need some use for it and also some testing.

Turbulent Schmidt number is definitely used

Reply:
Yes, we do not use it currently. I'd say we should use laminar Schmidt number when we use the laminar Prandtl number for heat transfer, to have consistent modeling for energy and species transport.